iOS/MaterialComponents/source/MaterialComponents/Extensions.cs

```diff
@@ -35,20 +113,22 @@ public static class MaterialFeatureHighlightStrings {
 	}
 
 	public partial class CardScheme {
-		SemanticColorScheme semanticColorScheme;
 		public SemanticColorScheme SemanticColorScheme {
 			get {
-				if (semanticColorScheme == null)
-					semanticColorScheme = Runtime.GetNSObject<SemanticColorScheme> (ColorScheme.Handle, false);
-
-				return semanticColorScheme;
+				return Runtime.GetNSObject<SemanticColorScheme> (ColorScheme.Handle, false);
 			}
 			set {
-				semanticColorScheme?.Dispose ();
-				semanticColorScheme = null;
 				ColorScheme = value;
 			}
 		}
+		//public ShapeScheme ShapeScheme {
```

Comment:
I couldn't think of another name for this - the types have changed, but are just different with a new `I`